tools/c7n_azure/tests/test_app_service_plan.py

```diff
@@ -46,71 +48,108 @@ def test_app_service_plan_schema_validate(self):
             self.assertTrue(p)
 
     @arm_template('appserviceplan.json')
-    def test_find_by_name(self):
+    def test_resize_plan_win(self):
```

Comment:
You won't be able to run this test twice, right?

Reply:
Correct. You would need to re-provision or set it up again.

Reply:
That isn't ideal   ;)   Took forever to fix all those last time.   I'd probably make/edit/fix resource in the setup/teardown (like the old test-storage-firewall-actions does).  Or just patch out the actual HTTP calls.